src/assets/QuillLocalAsset.php

```diff
@@ -16,14 +16,23 @@
 class QuillLocalAsset extends AssetBundle
 {
     /**
-     * @var string
+     * {@inheritDoc}
      */
-    public $sourcePath = '@bizley/quill/assets/quill';
+    public $sourcePath = '@npm/quill/dist';
 
     /**
-     * @var array
+     * {@inheritDoc}
      */
-    public $js = ['quill.min.js'];
+    public $js = [
+        'quill.min.js'
+    ];
+
+    /**
+     * {@inheritDoc}
+     */
+    public $css = [
+        'theme' => 'quill.core.css'
```

Comment:
Why is `theme` here?

Reply:
To address this css file directly on line **50**, **54**, or **58** without breaking possible configuration overrides